README.md

```diff
@@ -1,2 +1,62 @@
 # pt-salary-calculator
-Part-Time Salary Calculator is created for my own personal usage to help keep track of my part time earnings and job details from current, past and potential job opportunities.
+
+Part Time Salary Calculator helps centralise and track my part-time earnings and job details from my current and past jobs all into one website. This will ease the need of attempting to update the formulus in Google Sheets consistently and adding new tabs for new jobs taken.
```

Comment:
There is a spelling error for `formulus` at 
`...attempting to update the formulus in Google Sheets...`